build.cmd

```diff
@@ -0,0 +1,77 @@
+@echo off
+setlocal
+
+:Build
+
+echo **********starting to build the project**********
+set MSBuild="%ProgramFiles(x86)%\MSBuild\12.0\Bin\MSBuild.exe"
+if not exist %MSBuild% @set MSBuild="%ProgramFiles%\MSBuild\12.0\Bin\MSBuild.exe"
+
+set BUILDLOG=BuildResult.txt
+
+:BuildFull
+echo *** Build Microsoft.Odata.Full ***
+
+%MSBuild% sln/Microsoft.OData.Full.sln /m /nr:false  /p:Platform="Any CPU" /p:Desktop=true /v:M /fl /flp:LogFile=bin\msbuild.log;Verbosity=Normal > BUILDLOG

```

Comment:
if try this with clean clone branch, it would fail with 'bin' folder not created. Consider move the log to top folder?